tests/api_tests/v1/tests/api_tests/test_no_model_feature.py

```
@@ -205,7 +205,7 @@ def test_no_model_work_with_custom_vectors_in_search(self):
                                                          context={"tensor": [{"vector": [1, ] * self.DIMENSION,
                                                                               "weight": -1},
                                                                              {"vector": [1, ] * self.DIMENSION,
-                                                                              "weight": 1}], })
```

Maybe add a test to ensure a proper error is raised?

---

not sure what you mean. this test isn't supposed to raise an error, I just changed it because it now accidentally makes the context tensor a zero magnitude vector, which raises an error. I'll add the following tests to integration tests:
1. context tensors that cancel each other out (weights opposite of each other) to make a zero magnitude vector should raise an error.
2. search with context tensors with all zero weight should raise an error
3. search with context documents with all zero weight should raise an error